pom.xml

```diff
@@ -635,6 +635,52 @@
                     <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>${exec-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>create-licenses</id>
+                        <configuration>
+                            <mainClass>io.confluent.licenses.LicenseFinder</mainClass>
+                            <arguments>
+                                <!-- Note use of development instead of package so we pick up all dependencies.
+                                     This assumes both subprojects will be packaged together. -->
+                                <argument>-i ${project.build.directory}/${project.build.finalName}-package/share/java/acl</argument>
+                                <argument>-o ${project.parent.basedir}/licenses/overrides</argument>
+                                <argument>-f</argument>
+                                <argument>-h ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/licenses.html</argument>
+                                <argument>-l ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/licenses</argument>
+                                <argument>-n ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/notices</argument>
+                                <argument>-t ${project.name}</argument>
+                                <argument>-x connect-api-${ce.kafka.version}.jar,connect-json-${ce.kafka.version}.jar,connect-runtime-${ce.kafka.version}.jar,control-center-${io.confluent.blueway.version}.jar,licenses-${io.confluent.license-file-generator.version}.jar,confluent-licensing-new-${ce.kafka.version}.jar,control-center-frontend-${io.confluent.blueway-fe.version}.jar,confluent-command-${io.confluent.blueway.version}.jar</argument>
```

Comment:
@ewencp Just cycling back to this PR. Seems the `-x` flag is used to exclude stuff?

I copied this snipped from `blueway` -- to I am wondering what we would want/need to exclude for ksqlDB ?
```
connect-api-${ce.kafka.version}.jar,
connect-json-${ce.kafka.version}.jar,
connect-runtime-${ce.kafka.version}.jar,
control-center-${io.confluent.blueway.version}.jar,
licenses-${io.confluent.license-file-generator.version}.jar,
confluent-licensing-new-${ce.kafka.version}.jar,
control-center-frontend-${io.confluent.blueway-fe.version}.jar,
confluent-command-${io.confluent.blueway.version}.jar
```

Reply:
It's a list of packages that are also just under the confluent (enterprise) license, which is actually located elsewhere. So basically just proprietary ones, which shouldn't exist for this repo.

Generally the easiest way to generate the list though is to look at the packaged jars and filter out anything that is proprietary or belongs to this project anyway (i.e. you don't need to extract licenses from ksql jars since they are covered by the top-level LICENSE file within the package).